code/__DEFINES/jobs.dm

```diff
@@ -109,69 +144,116 @@
 #define DEPT_BITFLAG_MED (1<<6)
 #define DEPT_BITFLAG_SEC (1<<7)
 #define DEPT_BITFLAG_VIP (1<<8)
-#define DEPT_BITFLAG_SILICON  (1<<9)
-// should check the ones in `\_DEFINES\economy.dm`
-// It's true that bitflags shouldn't be separated in two DEFINES if these are same, but just in case the system can be devided, it's remained separated.
+#define DEPT_BITFLAG_SILICON  (1<<9) // currently not used, but remained here just in case someone wants it
 
 //-------------------------------------------------------------------------------------------
 //------------------------------------- Job names -------------------------------------------
 //-------------------------------------------------------------------------------------------
+// ****** WARNING:
+// 		DO NOT CHANGE KEY. You are fine to change NAME defines, but DO NOT DO to KEY defines
+#define JOB_UNASSIGNED "Unassigned"
+#define JOB_DEMOTED "Demoted"
```

Comment:
these are not real jobs that don't have to separate into key/datum.
and custom jobs will mean nothing because you can give multiple/single job flags if you want customs with different station role name. a custom special key will cause an runtime error.